src/domains/transaction/components/Transactions/Transactions.tsx

```diff
@@ -148,6 +149,14 @@ export const Transactions = memo(function Transactions({
 		return !hasEmptyResults || hasFilter || hasMore;
 	}, [activeMode, hasEmptyResults, activeTransactionType, isLoadingTransactions, hasMore]);
 
+	const enableFilter = useMemo(() => {
+		if (isLoadingTransactions || hasFilter) {
```

Comment:
We want to enable filter when transactions are loading or an active filter applied - I mean by that is initially we have all possible options selected this should be understood as no filters applied, when user picks a filter it means there are at least one transaction to display, so we shouldn't disable the element.